site/docs/v1/tech/apis/_application-response-body.adoc

```diff
@@ -251,6 +266,9 @@ The id of the Key used to sign the SAML response.
 [field]#application.samlv2Configuration.logoutURL# [type]#[String]# [since]#Available since 1.6.0#::
 The URL that the browser is taken to after the user logs out of the SAML service provider.
 
+[field]#application.samlv2Configuration.requireSignedRequests# [type]#[Boolean]# [optional]#Optional# [default]#Defaults to `false`# [since]#Available since 1.20.0#::
+Set this parameter equal to `true` to require the SAML v2 Service Provider to sign the request. When this value is `true` all requests missing a signature will be rejected and [field]#application.samlv2Configuration.defaultVerificationKeyId# is required.
```

Comment:
This should describe the behavior, but not describe how to set it since this is the response doc.

Reply:
Done.